### PR TITLE
Added zenodo json

### PR DESCRIPTION
**Pull request by DunklesArchipel**
I added all contributors up until now in alphabetical order for publishing on zenodo. I hope this is fine for you.

### CODE REVIEW

File: zenodo.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "name": "Hörmann, Nicolas",
+            "affiliation": "Fritz-Haber-Institut"
+        },
+        {   
+            "orcid": "0000-0002-3930-9107",
+            "name": "Rüth, Julian"
+        }
+        {   
+            "orcid": "0000-0003-2684-3711",
+            "name": "Technische Universität München",
```
Comment (saraedum) [inline zenodo.json:24]: That's a weird name ;)

File: zenodo.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "affiliation": 
+        }
+    ],    
+
+    "license": "GPL-3.0-or-later",
+    "title": "echemdb",
+    "description": "<p>The echemdb is supposed to be a repository for high quality data on electrochemical systems including a Python API to interact with the repository.</p>"
```
Comment (saraedum) [inline zenodo.json:31]:
```suggestion
    "description": "<p>The echemdb is aims to be a repository for high quality data on electrochemical systems. It provides a Python API to interact with the repository.</p>"
```

**Review by saraedum — APPROVED** [review]

File: zenodo.json (hunk 1 above)
Comment (DunklesArchipel) [reply zenodo.json:24]: it's impressive how I messed this up...

File: echemdb/data/local.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def collect_datapackages(data):
 
     EXAMPLES::
 
-        >>> packages = collect_datapackages(".")
+        >>> packages = collect_datapackages("data")
```
Comment (DunklesArchipel) [inline echemdb/data/local.py:33]:
```suggestion
        >>> packages = collect_datapackages(".")
```

**Pull request merged**